### DISCUSSION TITLE
motherboard GL12CP

### DISCUSSION THREAD

**Discussion by sancheznfy** [xCuri0/ReBarUEFI#61] [Q&A] [answered]
Hi
Any boddy tested GL12CP to enable ReBAR?

**Comment by sancheznfy** [comment]
![image](https://github.com/xCuri0/ReBarUEFI/assets/136899664/dd16e4fe-ee8c-4e4a-ad48-7571fe539e6b)
 Any other asus prime with H310 chipset

**Comment by sancheznfy** [comment]
![image](https://github.com/xCuri0/ReBarUEFI/assets/136899664/c21705cf-7f25-4b81-8b25-5c3480066f8e)
i try to enable for use with Radeon 6600

**Comment by xCuri0** [comment]
Try yourself and see there's no reason it wouldn't work

**Reply by sancheznfy** [reply]
i dont really understand, how to use your aplication

**Reply by sancheznfy** [reply]
Why did you close?

**Reply by xCuri0** [reply, marked as answer]
It's not a valid question, if you go to the list of working motherboards you can see most people are the first to try on their motherboard.


You can just try on your system and if there are any legitimate issues I might help with that
